### PR TITLE
Update README based on changes to the Public docs

### PR DESCRIPTION
**Pull request by GinoVin**
Review by ; ASAP
Based on the updated handled in this [JIRA](https://coveord.atlassian.net/browse/DOC-12560) to the updated [Log commerce events](https://docs.coveo.com/en/3188/coveo-for-commerce/log-commerce-events) articles. Some of the links in the read.me file needed to be updated. 
More updates to come

* If I messed up the PR template here, let me know what is missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,22 +100,16 @@ You should be able to observe the click event being transmitted to the Coveo bac
 
 ## Sending commerce specific events
 
-Commerce specific events such as product selections, shopping cart modifications and transactions are sent to Coveo in the compact [collect protocol](https://docs.coveo.com/en/l41i0031/build-a-search-ui/log-collect-events). Rather than explicitly assembling these payloads by hand, the eCommerce plugin provides apis to assemble and transmit the payloads. There are two event names that are specific to the eCommerce plugin:
+Commerce specific events such as product selections, shopping cart modifications and transactions are sent to Coveo in the compact [collect protocol](https://docs.coveo.com/en/l41i0031/build-a-search-ui/log-collect-events). Rather than explicitly assembling these payloads by hand, the eCommerce plugin provides APIs to assemble and transmit the payloads through events assembled through different plugin actions, or [pageviews](https://docs.coveo.com/en/l3am0254/coveo-for-commerce/page-views) automatically populated with the current page context.
```
Comment (fvanham-coveo) [inline README.md:103]: The `, or [pageviews]` is a bit of a run-on sentence here. Not sure what you're trying to say :)

File: README.md [hunk 2]
```diff
@@ -100,22 +100,16 @@ You should be able to observe the click event being transmitted to the Coveo bac
...
--   An [addition to the cart](https://docs.coveo.com/en/l3jg0266/coveo-for-commerce/measure-cart-page-events#measure-an-increase-in-item-quantity-in-cart)
--   A [removal from the cart](https://docs.coveo.com/en/l3jg0266/coveo-for-commerce/measure-cart-page-events#measure-a-decrease-in-item-quantity-in-cart)
+-   A [product detail view](https://docs.coveo.com/en/l2pd0522)
+-   [Cart modification events](https://docs.coveo.com/en/n39h1594/)
 -   A [cart purchase](https://docs.coveo.com/en/l39m0327/coveo-for-commerce/measure-a-purchase)
 -   An [event on a search-driven listing-page](https://docs.coveo.com/en/l41a1037/coveo-for-commerce/measure-events-on-a-listing-or-search-page)
 
-As a sample, here is how an [addition to the cart interaction](https://docs.coveo.com/en/l3jg0266/coveo-for-commerce/measure-cart-page-events#measure-an-increase-in-item-quantity-in-cart) is measured:
+As a sample, here is how an [cart modification event](https://docs.coveo.com/en/l3jg0266/coveo-for-commerce/measure-cart-page-events#measure-an-increase-in-item-quantity-in-cart) is measured:
```
Comment (fvanham-coveo) [inline README.md:112]:
```suggestion
As a sample, here is how a [cart modification event](https://docs.coveo.com/en/l3jg0266/coveo-for-commerce/measure-cart-page-events#measure-an-increase-in-item-quantity-in-cart) is assembled:
```

**Review by fvanham-coveo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lbourdages) [reply README.md:103]: page views are still events from our point of view 🤷🏼

**Review by lbourdages — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (GinoVin) [reply README.md:103]: I'll revert this change

**Pull request merged**